data_access_service/config/batch/generatecsv/compute_environment.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Now we do not necessary generate csv so it is better to remove name target csv

Also I see that during job submit, we can override some of the setting so we just need some very minimal setting here